code/src/App.js

```diff
@@ -1,9 +1,71 @@
-import React from 'react'
+import React, { useEffect, useState } from "react";
+
+import ThoughtForm from "./components/ThoughtForm";
+import ThoughtItem from "./components/ThoughtItem";
+import LoadingItem from "./components/Loading";
+
+import { API_URL, LIKES_URL } from "./utils/urls";
 
 export const App = () => {
+  const [thoughts, setThoughts] = useState([]);
+  const [newThought, setNewThought] = useState("");
+  const [loading, setLoading] = useState(false);
+
+  useEffect(() => {
+    fetchThoughts();
+  }, []);
+
+  const fetchThoughts = () => {
+    setLoading(true);
```

Comment:
Good that you use the loading component here maybe set the same backgrouns as the one you have in your main application :)